frontend/javascripts/oxalis/view/components/border_toggle_button.tsx

```diff
@@ -46,7 +46,9 @@ function BorderToggleButton(props: Props) {
         onClick={(event) => {
           if (event != null) {
             // @ts-expect-error ts-migrate(2339) FIXME: Property 'blur' does not exist on type 'EventTarge... Remove this comment to see the full error message
-            event.target.blur();
```

Comment:
I think this blur() can be removed, as it probably doesn't do anything. As far as I know, only one blur will have an effect, since only one element can have the page focus. I also just tested it and the second blur seems to be enough to solve the issue :tada:

Reply:
ah, it seems like it depends on where the user clicks exactly. sometimes the inner element gets the focus and sometimes the outer element. this is why both blurs make sense (however, with a slightly differing explanation than in your PR :)).